docs/networking/split-horizon_scenarios.md

```diff
@@ -0,0 +1,95 @@
+| **Namespace**                 | Kubernetes namespace to which this config is applicable. Defined by the user during config creation.                 |
+| **Base domain name**          | Domain name suffix used for generating full domain names for each Pod in the Replica Set. This field is also used as the **Split-Horizon DNS config** name. |
+| **TLS certificates**          | TLS certificate files for the provided base domain. See [instructions](https://docs.percona.com/percona-operator-for-mongodb/tls-manual.html#__tabbed_1_1). <br>- `cert.pem` - certificate<br>- `cert-key.pem` - certificate key<br>- `ca.pem` - certificate authority file |
+
+Once all values are entered, the Percona Everest administrator saves the configuration in the system.
+
+
+## Protect Split-Horizon DNS config with RBAC
+
+A Percona Everest administrator may want to restrict access to the Split-Horizon DNS configuration or the ability to apply a specific Split-Horizon DNS configuration for certain Percona Everest users. To achieve this, the administrator utilizes Percona Everest RBAC to define policies that restrict access to the selected instances. These RBAC policies apply to the entire Split-Horizon DNS configuration.
+
+### Limit access to Split-Horizon DNS config
+
+A Percona Everest administrator may want to allow specific users (for example, Alice) to fully manage (create, edit, read, delete) Split-Horizon DNS configurations.
+Other users can only read the Split-Horizon DNS configs and use them in database clusters, but cannot make any changes.
+
+To implement this, the Percona Everest administrator creates an RBAC policy:
+
+```sh
+p, alice, engine-features/split-horizon-dns-config, *, */*
+p, role:team-dev, engine-features/split-horizon-dns-config, read, */*
+```
+
+### Limit usage of Split-Horizon DNS config by users
+
+A Percona Everest administrator may want to allow a specific user (for example, bob) to use a particular Split-Horizon DNS config (for example, `mycompanydns` in namespace `ns-1`).
+All other users are not permitted to use this configuration. That is, they cannot see it in the system and cannot assign it to any database clusters they create or modify.
+
+To implement this, the Percona Everest administrator creates an RBAC policy:
+
+```sh
+p, bob, engine-features/split-horizon-dns-config, read, ns-1/mycompanydns
+```
+
+## Apply Split-Horizon DNS config to new database cluster
+
+!!! info "important"
+    - Only one **Split-Horizon DNS config** can be applied to one database Cluster.
+    - **Split-Horizon DNS** feature can be enabled only if **Sharding** is disabled.
+
+When creating a new cluster for Percona Operator for MongoDB, you can choose to enable the Split-Horizon DNS feature.
+
+The system provides an additional **Split-Horizon DNS** panel on the **Advanced Configuration** step of the database cluster creation wizard. By default, the Split-Horizon DNS feature toggle is disabled.
+
```

Comment:
the `Advanced configuration` panel screenshot will be useful here.

Reply:
Fixed